lib/ukdebug/gdbstub.c

```diff
@@ -1106,3 +1113,78 @@ int gdb_dbg_trap(int errnr, struct __regs *regs)
 
 	return r;
 }
+
+static int gdb_setup_backing_console(void)
+{
+	/* Ignore breakpoints if the stub is not enabled on the command line. */
+	if (gdb_backing_console_id == GDB_BACKING_CONSOLE_NONE) {
+		uk_pr_info("Backing console not set\n");
+		return -ENXIO;
```

Comment:
I would not consider this as error case.

Reply:
I think it doesn't much matter here. Just look at the caller code of this function. I could also return a positive value instead of `-ENXIO` here. But we can't return `0` as there isn't actually a console that can be used.

Reply:
Hum... Maybe we can get it clearer if we move this check to `gdb_late_breakpoint()`, before calling this function. `gdb_backing_console_id == GDB_BACKING_CONSOLE_NONE` is saying that we do not want to have gdb stub waiting, correct?

Reply:
Yes, that's correct